pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -401,10 +404,33 @@ public void start() throws PulsarServerException {
             this.serviceUnitStateChannel.listen(splitManager);
             this.leaderElectionService.start();
             pulsar.runWhenReadyForIncomingRequests(() -> {
-                try {
-                    this.serviceUnitStateChannel.start();
-                } catch (Exception e) {
-                    failStarting(e);
+                int retry = 0;
+                while (!Thread.currentThread().isInterrupted()) {
+                    try {
+                        brokerRegistry.register();
+                        this.serviceUnitStateChannel.start();
+                        break;
+                    } catch (Exception e) {
+                        log.warn("The broker:{} failed to start service unit state channel. Retrying {} th ...",
+                                pulsar.getBrokerId(), ++retry, e);
+                        try {
+                            TimeUnit.SECONDS.sleep(Math.min(retry * 10L, MAX_ROLE_CHANGE_RETRY_DELAY_IN_MILLIS));
```

Comment:
There's currently the org.apache.pulsar.common.util.Backoff class for doing exponential backoff with jitter. I think it would be better to use that solution instead of adding another way to do backoffs unless there's a specific reason for this logic.

Reply:
I want to set the max retry count, but since like the `Backoff` doesn't support it.

Reply:
@Demogorgon314 Could you use the Backoff class for the delay calculation and keep the existing logic for max retry count?

Reply:
Sure, it was updated.